### PR TITLE
Update readme with requirements and more sass help

### LINKED ISSUE #53: README should contain install instructions
When running `make run` on a fresh machine I get:

``` bash
$ make run
docker-compose up -d
#ERROR: Couldn't connect to Docker daemon. You might need to install Docker:

http://docs.docker.io/en/latest/installation/ubuntulinux/
Makefile:45: recipe for target 'run' failed
make: *** [run] Error 1
```

There should be instructions on how to set up the full environment in the README file.

### PR DESCRIPTION
**Pull request by pmahnke**
## Done

* added a section on requirements for Docker and npm
* added more details on sass errors

## QA

1. run through the install from scratch, see if they help

## Issue / Card

Fixes #53

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -23,13 +23,30 @@ Now visit <http://127.0.0.1:8001>
...
 
 If you get an ubuntu-vanilla-theme error on first startup, go into your /node_modules/ubuntu-vanilla-theme directory and run:
 
 ``` npm install --save-dev gulp-sass@2 ```
 
+then
+
+``` docker-compose run sass ```
+
+and even possibly
```
Comment (deadlight) [inline README.md]: Surely this either needs to happen or not.

Comment (pmahnke) [inline README.md]: @deadlight I don't know.. we could belt & braces make everyone do both?  I don't mind

Comment (pmahnke) [inline README.md]: @nottrobin or @WillMoggridge do you perhaps know the answer here?

Comment (pmahnke) [inline README.md]: @nottrobin or @WillMoggridge do you perhaps know the answer here?

Comment (WillMoggridge) [inline README.md]: The above command should be enough to sort out the sass and this command won't be needed.
I am for expanding and explaining that this command can optionally restart the stack if you continue to have a problem or other problems.

Comment (pmahnke) [inline README.md]: @WillMoggridge I can confirm that the NPM command was not enough...

**Pull request merged**